spring-examples/spring-session/simple-spring-session-client-server/README.md

```diff
@@ -0,0 +1,45 @@
+# Simple Spring Session Client-Server Example
+
+## Description
+This example is a simple Spring Session example, where there is a GemFire client initialized within a Spring Boot web app, that will support session information management. This example is setup to expire session information within 30 second of it being committed. This 30 second expiration is for demonstration purposes only.
+
+## Prerequisites
+In order to run the following example one requires the following to be set up:
+* Java 17
+* Latest VMware GemFire is installed locally
+* Set up local `gradle` or `maven` environment by following Steps 1-6 and 8 from [Spring Boot For VMware GemFire Quickstart](https://docs.vmware.com/en/Spring-Boot-for-VMware-GemFire/index.html#spring-boot-for-vmware-gemfire-quick-start-0)
+
+## Steps to run
+1. Clone `Simple Spring Session Client Server` project
+2. Start GemFire server-side processes 
+   1. Open new Terminal
+   2. `cd` into the `Simple Spring Session Client Server` project directory
+   3. Start `gfsh` located in the `{VMware_GemFire_installation_location}/bin` directory
+   4. Locate the `start.gfsh` file in the `Simple Spring Session Client Server` project
+   5. Launch the Locator and Server by running `start.gfsh` file located in the `Simple Spring Session Client Server` project directory in `gfsh`. <br> e.g `run --file={pathToStartGfshFile}/start.gfsh`
+   6. Confirm that the locator and server have started and configured correctly, by confirming that the status of the commands run are `Status : PASSED` 
+   7. Confirm that the locator and server are running by running `list members`, which should contain two entries
+   8. Confirm the session information region has been created correctly, by running `list regions` which should contain a Region entry `ClusteredSpringSessions`
+3. Start Client
+   1. Open a new Terminal
+   2. `cd` into the `Simple Spring Session Client Server` project directory
+   3. Run `./gradlew clean tomcatRun`
```

Comment:
This isn't bootRun?

Reply:
No.. this starts a boot app in a Tomcat env... pretty cool actually